style.css

```diff
@@ -924,4 +1042,131 @@ footer img {
   .form-li input {
     width: 684px;
   }
+
+  .ul-listing {
+    list-style: none;
+    display: flex;
+    gap: 40px;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  /* Sidebar */
+
+  .menu {
+    width: 14px;
+    height: 14px;
+    background: transparent;
+    border-color: transparent;
+    color: #fff5e1;
+    transition: all 0.3s linear;
+    cursor: pointer;
+    animation: bounce 2s ease-in-out infinite;
+  }
+
+  .menu:hover {
+    color: var(--clr-primary-7);
+  }
+
+  @keyframes bounce {
+    0% {
+      transform: scale(1);
+    }
+
+    50% {
+      transform: scale(1.5);
+    }
+
+    100% {
+      transform: scale(1);
+    }
+  }
+
+  .close-menu {
+    background: transparent;
+    border-color: transparent;
+    color: #fff5e1;
+    transition: all 0.3s linear;
+    cursor: pointer;
+    display: flex;
+    height: 20px;
+    width: 20px;
+    left: 6px;
+    top: 6px;
+    border-radius: 0;
+    margin-top: 30px;
+  }
+
+  .sidebar {
+    position: absolute;
+    height: 755px;
+    width: 100%;
+    border-radius: 0;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 57px;
+    background-color: #3c3a39;
+    row-gap: 1rem;
+
+    /* box-shadow: #3c3a39; */
```

Comment:
- [ ] Please, remove this piece of old commented-out code to improve the cleanliness of the project. 💪